runtime/vm/jvminit.c

```diff
@@ -6368,19 +6368,19 @@ protectedInitializeJavaVM(J9PortLibrary* portLibrary, void * userData)
 			 * as "size_t" type.
 			 */
 
-			j9tty_printf(PORTLIB, "[Global flags]\n");
+			j9file_printf(PORTLIB, J9PORT_TTY_OUT, "[Global flags]\n");
```

Comment:
Can you add a comment here in the code explaining why this option is different and prints to `stdout` when everything else goes to `stderr`?

I have reservations about making this change as it contradicts all of our existing options direction.

Reply:
There isn't really an alternative if we want to support PrintFlagsFinal: the output has been produced by Hotspot for many years on `stdout` and is expected on `stdout`.

elasticsearch specifically does not see it if it's produced on `stderr`.

Reply:
I've added a new line to the main comment that introduces the output that's desired.

Reply:
Understood.  I dislike adding the inconsistency in our output strategy as it means understanding and explaining when it's OK to send to `stdout` rather than `stderr`.  The comment you've added helps to clarify that.